prijateli_tree/app/templates/start_of_game.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+{% block title %}
+  Start of Game
+{% endblock title %}
+{% block content %}
+  <div class="container-sm general-container">
```

Comment:
At some point, it'd be nice to have a template that doesn't always require this div, but that's a very later nice to have.